dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.messaging.internal-3.1.feature

```diff
@@ -2,7 +2,7 @@
 symbolicName=io.openliberty.messaging.internal-3.1
 singleton=true
 IBM-App-ForceRestart: uninstall
-IBM-API-Package: jakarta.jms; version="3.0"; type="spec"
+IBM-API-Package: jakarta.jms; version="3.1"; type="spec"
```

Comment:
We should not put the version attribute on the `IBM-API-Package`.  It is not used at runtime and only leads to inconsistencies when we forget to update the version here.  The version for the packages come from their source bundle exporter.